src/main/java/com/splunk/logging/HttpEventCollectorLog4jAppender.java

```diff
@@ -38,6 +38,7 @@
 public final class HttpEventCollectorLog4jAppender extends AbstractAppender
 {
     private HttpEventCollectorSender sender = null;
+    private boolean sendFullThrowable = false;
```

Comment:
It seems better to disambiguate from the current behaviour of sending the throwable as the message alone.